### PR TITLE
AppVeyor: Use existing Go 1.6 instead of downloading + installing

### PR DESCRIPTION
**Pull request by zanderz**
@maxtaco

### CODE REVIEW

**Comment by oconnor663** [comment]
Should we go ahead and update to 1.6? We've already done it on Linux. (And OSX?)

**Comment by zanderz** [comment]
I have no issues with 1.6

File: appveyor.yml [hunk 1]
```diff
@@ -4,16 +4,9 @@ environment:
     GOPATH: c:\work\
     GO15VENDOREXPERIMENT: 1
```
Comment (oconnor663) [inline appveyor.yml]: We can kill this now if we want I think?

**Comment by oconnor663** [comment]
If we wanted to do anything complicated with specific versions of Go, we could consider testing [many of them at once](https://github.com/oconnor663/duct/blob/master/appveyor.yml#L4) :-D This probably matters more on the Linux side of things, though, where kind of care about Arch users building the client on their own machines.

**Comment by taruti** [comment]
Updating to Go 1.6 would be very nice.

**Comment by zanderz** [comment]
On the 6th or 7th try, it passed without modification. Merging.

**Pull request merged**